pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -207,7 +207,7 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 		if kc.CrashLoopBackOff.MaximumBackOffPeriod == nil {
 			allErrors = append(allErrors, fmt.Errorf("invalid configuration: FeatureGate KubeletCrashLoopBackOffMax is enabled, CrashLoopBackOff.MaximumBackOffPeriod must be set"))
 		}
-		if kc.CrashLoopBackOff.MaximumBackOffPeriod != nil && utilvalidation.IsInRange(int(kc.CrashLoopBackOff.MaximumBackOffPeriod.Duration), int(time.Second), int(300*time.Second)) != nil {
+		if kc.CrashLoopBackOff.MaximumBackOffPeriod != nil && utilvalidation.IsInRange(int(kc.CrashLoopBackOff.MaximumBackOffPeriod.Duration.Seconds()), 1, 300) != nil {
```

Comment:
Does this allow 300.9 ?  Is that a problem?

Reply:
I don't think it's a big problem, but I've tightened it up.

Reply:
In 499e95b